.maestro/setup_ui_tests.sh

```diff
@@ -78,6 +79,8 @@ while [[ "$#" -gt 0 ]]; do
     case $1 in
         --skip-build) 
             skip_build=1 ;;
+        --rebuild) 
+            rebuild=1 ;;
```

Comment:
Support `--rebuild` arg so that you can make local changes while running the tests quicker.